src/os/windows/ottdres.rc.in

```diff
@@ -118,7 +118,7 @@ END
 #endif    // !_MAC
 
 #ifdef __MINGW32__
-1 24 "${CMAKE_SOURCE_DIR}/projects/dpi_aware.manifest"
+1 24 "${CMAKE_SOURCE_DIR}/projects/openttd.manifest"
```

Comment:
Seems like os/windows/ is a better location now :)

Reply:
I will move it then, luckily only one place needs to be touched for that in cmake script ;)